rules/windows/exfiltration_winrar_encryption.toml

```diff
@@ -0,0 +1,38 @@
+[metadata]
+creation_date = "2020/12/04"
+maturity = "production"
+updated_date = "2020/12/04"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies use of winrar.exe to create an encrypted rar file using the -hp option."
```

Comment:
update description and rule name to reflect use of compression utility (7z and rar)